worlds/witness/options.py

```diff
@@ -424,7 +429,7 @@ class TheWitnessOptions(PerGameCommonOptions):
     panel_hunt_postgame: PanelHuntPostgame
     panel_hunt_discourage_same_area_factor: PanelHuntDiscourageSameAreaFactor
     early_caves: EarlyCaves
-    early_symbol_item: EarlySymbolItem
+    early_good_items: EarlyGoodItems

```

Comment:
Should `early_symbol_item` be marked removed?

Reply:
Actually, did this even exist for a version? 🤔

Reply:
Addressed